src/apps/dashboard/controllers/networking.js

```diff
@@ -142,6 +146,14 @@ export default function (view) {
         page.querySelector('#chkEnableIP6').checked = config.EnableIPv6;
         page.querySelector('#chkEnableIP4').checked = config.EnableIPv4;
         page.querySelector('#txtPublishedServer').value = (config.PublishedServerUriBySubnet || []).join(', ');
+        const selectPublicUserListing = page.querySelector('#selectPublicUserListing');
+        if (config.PublicUserListingLocalOnly && config.PublicUserListing) {
+            selectPublicUserListing.value = 'local';
+        } else if (config.PublicUserListing) {
+            selectPublicUserListing.value = 'always';
+        } else {
+            selectPublicUserListing.value = 'never';
+        }
```

Comment:
I would put a blank line here as well to separate the `if/else` block (activity 1) from hiding the loading (activity 2).
```suggestion
        }

```
Iirc, we have a rule to separate blocks in general, but it doesn't seem to be checked by linter. 🤷‍♂️